DATM/AtmBundleCreate.F90

```diff
@@ -87,7 +89,7 @@ subroutine AtmBundleCreate(gcomp,importState, exportState, rc)
 
    write(msgString,'(i4,2a14,a16,2a14,a16)')ii,' added field ',trim(fnameb),' to AtmBundleBak', &
                                                '   and field ',trim(fnamef),' to AtmBundleFwd'
-   call ESMF_LogWrite(trim(msgString), ESMF_LOGMSG_INFO, rc=rc)
```

Comment:
Not sure why, but we may still want to have the return code?

Reply:
I removed all the return codes at the end of the LogWrite calls that were just for information messages vs an rc created by a call to an ESMF routine.